README.md

```diff
@@ -1,3 +1,6 @@
+![Static Badge](https://img.shields.io/badge/API%20coverage-0%25-red)
+![test workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/tests.yml/badge.svg)
+![quality workflow](https://github.com/scylladb-zpp-2024-javascript-driver/scylladb-javascript-driver/actions/workflows/code-quality.yml/badge.svg)
```

Comment:
😆

Reply:
Hmm, is this truly dynamic (depending on the status of checks on `main`) or are those static images? I'm suspicious about this laugh above :P

Reply:
I think in the current state it takes the last finished job from any branch

Reply:
But I don't see the reason for the first badge, as it appears to be a static one

Reply:
> I think in the current state it takes the last finished job from any branch

It definitely should only care about `main`.

Reply:
@PiotrJunior

Reply:
Because what point would be in displaying checks state of some new WIP branch?
I always the badge is to confirm the stable state of `main`.

Reply:
> But I don't see the reason for the first badge, as it appears to be a static one

I see no point in the first static badge, too.